src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/AbstractTypeImportCompletionService.CacheEntry.cs

```diff
@@ -23,16 +23,19 @@ internal abstract partial class AbstractTypeImportCompletionService
             public Checksum Checksum { get; }
 
             private ImmutableArray<TypeImportCompletionItemInfo> ItemInfos { get; }
+            private int PublicItemCount { get; }
```

Comment:
doc.

Reply:
Added doc. We need to exact number of items we are dealing with, since some of the optimization rely on this knowledge to avoid allocation from calls to `Resize`